src/codegen/templates/system/physical_channel.py.mako

```diff
@@ -34,12 +34,13 @@ class PhysicalChannel:
     """
     __slots__ = ['_name', '__weakref__']
 
-    def __init__(self, name):
+    def __init__(self, name, interpreter):
         """
         Args:
             name (str): Specifies the name of the physical channel.
         """
         self._name = name
```

Comment:
update docstring in all places according to new addition of params

Reply:
Updated docstring wherever applicable.